homework/Repository.js

```diff
@@ -13,8 +13,42 @@ class Repository {
    * @param {HTMLElement} container The container element in which to render the repository.
    */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.repository, null, 2));
+    const repo = Object.assign({}, this.repository, {
+      updated_at: new Date(this.repository.updated_at).toLocaleString(),
+    });
+
+    const repoTitles = { description: 'Description: ', forks: 'Forks: ', updated_at: 'Updated: ' };
+
+    const table = Util.createAndAppend('table', container, { class: 'table' });
+    const tbody = Util.createAndAppend('tBody', table);
+
+    const firstRow = Util.createAndAppend('tr', tbody);
+    let leftCell = Util.createAndAppend('td', firstRow);
+    let rightCell = Util.createAndAppend('td', firstRow);
+
+    Util.createAndAppend('span', leftCell, { text: 'Repository: ', class: 'repo-child left-cell' });
+    Util.createAndAppend('a', rightCell, {
+      text: this.repository.name,
```

Comment:
Instead of `this.repository` you could (for consistency) use `repo` here, which is an exact clone of `this.repository`, except for the formatted date.